papermill/execute.py

```diff
@@ -222,6 +218,39 @@ def _parameterize_notebook(nb, kernel_name, parameters):
     after = nb.cells[param_cell_index + 1:]
     nb.cells = before + [newcell] + after
 
+    nb.metadata.papermill['parameters'] = parameters
```

Comment:
What does this do? I once upon a time knew but have forgotten :-(

Stores the parameters in the metadata for later inspection?

Reply:
Yep. I don't know if we use it much, but in theory you could read another notebook and see how it got parameterized easily this way